fulp_modules/features/antagonists/bloodsuckers/code/monster_hunter/monsterhunter_datum.dm

```diff
@@ -15,6 +15,10 @@
 	var/give_objectives = TRUE
 	var/datum/action/bloodsucker/trackvamp = new /datum/action/bloodsucker/trackvamp()
 	var/datum/action/bloodsucker/fortitude = new /datum/action/bloodsucker/fortitude/hunter()
+	///the rabbit illusion trauma related to us
+	var/datum/brain_trauma/special/rabbit_hole/sickness
```

Comment:
This sells a monster hunter out, right?
Is it intentional? Can monster hunters be "cured"?